commit fc5d4c609bfcb272b523f9c35a210d4f906dbc9d
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-09-03T19:27:46+00:00 (1 year, 1 month ago)
    
    TOOLS-2073 Allow importing repositories with "." (dot) in their name into Gerrit

diff --git a/bin/crimport b/bin/crimport
index 5831976..ead9f5b 100755
--- a/bin/crimport
+++ b/bin/crimport
@@ -98,7 +98,7 @@ function main
 	fi
 
 	if [[ ! $1 =~ ^[-a-zA-Z0-9_]+$ ]] ||
-	   [[ ! $2 =~ ^[-a-zA-Z0-9_]+$ ]]; then
+	   [[ ! $2 =~ ^[-a-zA-Z0-9_.]+$ ]]; then
 		fail "GitHub account or repo name contains unsupported" \
 		    "characters"
 	fi
@@ -229,7 +229,7 @@ function igp_project_create
 function igp_github_clone
 {
 	if [[ ! $1 =~ ^[-a-zA-Z0-9_]+$ ]] ||
-	   [[ ! $2 =~ ^[-a-zA-Z0-9_]+$ ]]; then
+	   [[ ! $2 =~ ^[-a-zA-Z0-9_.]+$ ]]; then
 		fail "bogus account or repo name!"
 	fi
 
@@ -245,7 +245,7 @@ function igp_github_clone
 function igp_github_protect
 {
 	if [[ ! $1 =~ ^[-a-zA-Z0-9_]+$ ]] ||
-	   [[ ! $2 =~ ^[-a-zA-Z0-9_]+$ ]]; then
+	   [[ ! $2 =~ ^[-a-zA-Z0-9_.]+$ ]]; then
 		fail "bogus account or repo name!"
 	fi
 
